.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
 	"editor.formatOnSave": true,
 	"editor.codeActionsOnSave": {
 		"source.fixAll": true
-	}
+	},
+	"typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Is this really needed?

Reply:
It sounds like it's only needed if VS Code cannot otherwise detect what version of TypeScript to use; did you find it necessary to get 3.7 features to work?

Reply:
So this showed up when I selected on bottom right, where it has TS version, and changed from 3.6.3 ("What VSCode has") to 3.7.2 ("What the workspace has"). It may be until 1.41 that VSCode switches to 3.7?

Reply:
I tested it out, and we do in fact need this line or the Intellisense starts disagreeing with the actual build. Presumably, when VSCode moves to 3.7+, this would no longer be needed.